matson.theme

```diff
@@ -77,10 +77,18 @@ function matson_preprocess_field_field_s_news_date(&$variables, $hook) {
 }
 
 /**
- * Prepares non node/panel pages to be in a single column.
+ * Add 'earth-matters' class to all Earth Matters news items.
  */
 function matson_preprocess_page(&$vars) {
-  // Nothing to see here.
+  if(
+    isset($vars['node']) && 
+    $vars['node'] instanceof \Drupal\node\NodeInterface && 
+    $vars['node']->bundle() == 'stanford_news' &&
+    $vars['node']->get('field_s_news_category')->getString() == '81'
```

Comment:
Is there ever a scenario where a node is an earth mather and other categories? If so we might want to tweak this a little bit. if a node is tagged with three terms, the `getString()` will return `81,33,59` or something like that.

Reply:
hm, good question. Let me check.

Reply:
OK, I checked. It is rare that there are some that are double-tagged.  But there are some. Let me go ask the channel before we do anything here.

Reply:
Josie suggests that the ones that are double-tagged SHOULD NOT be double-tagged. So, I don't think we need to contend with those (and it might be best if nothing happened for those cases. )Thanks for raising this, @pookmish !

Reply:
Ok sounds good. Thanks for checking. If it does come up, we can adjust as needed.